### DISCUSSION TITLE
How to download model Oogabooga

### DISCUSSION THREAD

**Discussion by Katopr** [oobabooga/text-generation-webui#3367] [Q&A]
After opening "start_windows.bat" this shows up:
*******************************************************************
* WARNING: You haven't downloaded any model yet.
* Once the web UI launches, head over to the bottom of the "Model" tab and download one.
*******************************************************************

bin D:\gry\oogaboogawebui\oobabooga_windows\oobabooga_windows\installer_files\env\lib\site-packages\bitsandbytes\libbitsandbytes_cuda117_nocublaslt.dll
2023-07-30 08:40:08 INFO:Loading the extension "gallery"...
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.

For example, i would like to download pygmalion 13b (https://huggingface.co/PygmalionAI/pygmalion-13b).

Ik this question might sound stupid but its my first time trying out stuff like this.

**Comment by mfoldes** [comment]
I'll use [https://huggingface.co/PygmalionAI/pygmalion-2.7b/tree/main](https://huggingface.co/PygmalionAI/pygmalion-2.7b/tree/main) as an example.

You've gotten Gradio to load and that's step 1. The next step is to go over to the model tab and focus on what I've boxed in red boxes
![image](https://github.com/oobabooga/text-generation-webui/assets/6062263/6fa925d3-f533-4100-87dc-ffac9cb5017f)

The Download Model or LoRA uses download.py and the model you want can be downloaded by taking the PygmalionAI/pygmalion-2.7b part of the URL I used in the first sentence, entering it into the text box and selecting 'Download'
![image](https://github.com/oobabooga/text-generation-webui/assets/6062263/209983f1-5b18-49df-8cec-e782b7db9a94)

When it's done downloading, Go to the model select drop-down, click the blue refresh button, then select the model you want from the drop-down. Click load and the model should load up for you to use.
![image](https://github.com/oobabooga/text-generation-webui/assets/6062263/23ecef0c-8272-4663-82b3-f6a14b0b6e38)

Wait for the model to load and that's it, it's downloaded, loaded into memory and ready to go.

**Comment by missionfloyd** [comment]
The files in the Pygmalion 7B/13B repos are patches that need to be applied to Llama, they can't be used directly. Try these instead.
https://huggingface.co/notstoic/pygmalion-13b-4bit-128g
https://huggingface.co/TheBloke/Pygmalion-13B-SuperHOT-8K-GPTQ
